cmd/spiffe-helper/config/config.go

```diff
@@ -164,16 +174,61 @@ func (c *Config) ValidateConfig(log logrus.FieldLogger) error {
 		return errors.New("at least one of the sets ('svid_file_name', 'svid_key_file_name', 'svid_bundle_file_name'), 'jwt_svids', or 'jwt_bundle_file_name' must be fully specified")
 	}
 
+	if c.CertFileMode == "" {
+		c.CertFileMode = strconv.FormatUint(uint64(defaultCertFileMode), 8)
+	}
+	if c.KeyFileMode == "" {
+		c.KeyFileMode = strconv.FormatUint(uint64(defaultKeyFileMode), 8)
+	}
+	if c.JwtBundleFileMode == "" {
+		c.JwtBundleFileMode = strconv.FormatUint(uint64(defaultJwtBundleFileMode), 8)
+	}
+	if c.JwtSvidFileMode == "" {
+		c.JwtSvidFileMode = strconv.FormatUint(uint64(defaultJwtSvidFileMode), 8)
+	}
+
 	return nil
 }
 
 func NewSidecarConfig(config *Config, log logrus.FieldLogger) *sidecar.Config {
+	certFileMode := os.FileMode(defaultCertFileMode)
+	parsedCertFileMode, err := strconv.ParseUint(config.CertFileMode, 8, 32)
+	if err != nil || parsedCertFileMode > math.MaxUint32 {
+		log.WithError(err).Error("failed to parse file mode, using default")
+	} else {
+		certFileMode = os.FileMode(parsedCertFileMode) //nolint:gosec,G115
```

Comment:
I'm open to ideas on how to avoid disabling the linter rule on this line. But nothing I was trying was working.

Reply:
Fixed by avoiding string type

Reply:
The change away from string type was reverted. This error came back with the usage of int type (signed to unsigned conversion).

Reply:
Oh ok, I still see at `int` in the changeset.

Reply:
It is int. Which is why the `nolint` is added back as well.